tests/unit/components/Code.spec.ts

```diff
@@ -1,5 +1,7 @@
 import Code from "@/components/Code.vue";
 import { shallowMount } from "@vue/test-utils";
+import store from "@/store";
```

Comment:
This bit needs to be actually used.

Also please run tests and fix them. The Code component needs to switch between HTML and Markdown mode in the tests.

This file is failing.

This old code is like this:

```
      data() {
        return {
          asHtml: true,
        };
      },
```
 

I tried adding this and it didn't work.
```

    wrapper.setData({ asHtml: true })
    store.setAsHtml(true)
```